ct/5etools.sh

```diff
@@ -51,6 +51,10 @@ function update_script() {
         rm -rf "/opt/${APP}"
         mv "${APP}-src-${RELEASE:1}" "/opt/${APP}"
         mv "/opt/img-backup" "/opt/${APP}/img"
+        cd /opt/5etools
+        npm i
```

Comment:
```suggestion
        npm install
```

I would suggest using the fully named command instead of the alias.